packages/financial-templates-lib/src/price-feed/DefaultPriceFeedConfigs.js

```diff
@@ -384,9 +384,21 @@ const defaultConfigs = {
   CNYUSD: {
     type: "tradermade",
     pair: "CNYUSD",
-    minuteLookback: 7200, // interval=minute allowed lookback up to 2 days
-    hourlyLookback: 604800, // interval=hour allowed lookback up to 2 months
-    minTimeBetweenUpdates: 600
+    minTimeBetweenUpdates: 600,
+    // Since TradeMadeAPI timeseries data is unavailable over the weekend,
+    // set either the `minuteLookback` or the `hourlyLookback` to be longer than 3 days (259200 seconds)
+    // so that we can fetch the last known price prior to the weekend start.
+    // For example, if `minuteLookback = 7200`, then 2 hours after the weekend has "begun",
+    // `TraderMadePriceFeed.updateMinute()` will fail to find any prices. Simultaneously
+    // setting `hourLookback = 604800` ensures that if `updateMinute()` fails then `updateHourly()`
+    // will return the last hourly price. Alternatively, you can set `minuteLookback = 172800`, or
+    // 2 days (the maximum minute interval lookback) which will allow `updateMinute()` to fetch
+    // data for longer into the off-market period. We set `minuteLookback` to 7200 and `hourlyLookback`
+    // to 604800 so that we can both reduce the amount of data to parse from the minute-interval
+    // and ensure that we can always find a price when weekend prices are not available.
```

Comment:
small nit: Due to our styling guide, this can be represented with a few smaller lines if you try take each line to 120 chars.

```suggestion
// Since TradeMadeAPI timeseries data is unavailable over the weekend, set either the `minuteLookback` or the
// `hourlyLookback` to be longer than 3 days (259200 seconds) so that we can fetch the last known price prior to the
// weekend start. For example, if `minuteLookback = 7200`, then 2 hours after the weekend has "begun",
// `TraderMadePriceFeed.updateMinute()` will fail to find any prices. Simultaneously setting `hourLookback = 604800`
// ensures that if `updateMinute()` fails then `updateHourly()` will return the last hourly price. Alternatively, you
// can set `minuteLookback = 172800`, or 2 days (the maximum minute interval lookback) which will allow `updateMinute()`
// to fetch data for longer into the off-market period. We set `minuteLookback` to 7200 and `hourlyLookback` to 604800 so
// that we can both reduce the amount of data to parse from the minute-interval and ensure that we can always find a price when 
// weekend prices are not available.
```